api/src/main/java/org/pmiops/workbench/db/model/DbStorageEnums.java

```diff
@@ -4,17 +4,7 @@
 import com.google.common.collect.ImmutableBiMap;
 import org.pmiops.workbench.db.model.DbBillingProjectBufferEntry.BufferEntryStatus;
 import org.pmiops.workbench.db.model.DbWorkspace.BillingMigrationStatus;
-import org.pmiops.workbench.model.AnnotationType;
-import org.pmiops.workbench.model.Authority;
-import org.pmiops.workbench.model.BillingAccountType;
-import org.pmiops.workbench.model.BillingStatus;
-import org.pmiops.workbench.model.CohortStatus;
-import org.pmiops.workbench.model.Degree;
-import org.pmiops.workbench.model.EmailVerificationStatus;
-import org.pmiops.workbench.model.ReviewStatus;
-import org.pmiops.workbench.model.SpecificPopulationEnum;
-import org.pmiops.workbench.model.WorkspaceAccessLevel;
-import org.pmiops.workbench.model.WorkspaceActiveStatus;
+import org.pmiops.workbench.model.*;
```

Comment:
I think we decided as a team to not use wildcards in import statements. Assuming your using Intellij? Setting for this.